### PR TITLE
Feature/catd 133 android sdk can connect and fetch

### PR DESCRIPTION
**Pull request by Fabio-Insolia-LiveStyled**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
     <string name="ordering_journey">Ordering Journey</string>
+    <string name="campaign_automation">Campaign Automation</string>
     <string name="audio_guides_help">audioGuidesHelp</string>
     <string name="tandc">tAndC</string>
     <string name="privacy">privacy</string>
     <string name="about">about</string>
     <string name="about_company">aboutCompany</string>
     <string name="purchaseHelp">purchasesHelp</string>
-    <string name="query_web_page">Query Web Page</string>
+    <string name="query_web_page">Query Campaign Automation</string>
```
Comment (LokmaneKrizou) [inline app/src/main/res/values/strings.xml:23]: maybe we should use a different string for campaign automation button?

File: sdk/src/main/java/com/realifetech/sdk/campaignautomation/data/model/GetContentByExternalId.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.realifetech.sdk.campaignautomation.data.model
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class GetContentByExternalId(
+    val __typename: String,
```
Comment (LokmaneKrizou) [inline sdk/src/main/java/com/realifetech/sdk/campaignautomation/data/model/GetContentByExternalId.kt:8]: do we need `__typename ` field?

Comment (Fabio-Insolia-LiveStyled) [reply sdk/src/main/java/com/realifetech/sdk/campaignautomation/data/model/GetContentByExternalId.kt:8]: Truly ? no ... but I saw that is part of a lot of other calls so I included blindly
Will remove

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Fabio-Insolia-LiveStyled) [reply app/src/main/res/values/strings.xml:23]: Ops

**Pull request merged**